src/cryptoadvance/specter/util/wallet_importer.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, wallet_json, specter, device_manager=None):
         if device_manager is None:
             device_manager = specter.device_manager
         try:
-            self.wallet_data = json.loads(wallet_json)
+            self.wallet_data = json.loads("".join(wallet_json.split()))
```

Comment:
Maybe we can do it a bit more concise:
```suggestion
            self.wallet_data = json.loads(wallet_json.replace(" ", ""))
```